### PR TITLE
changes to bin.sh and docs

### PR DESCRIPTION
**Pull request by Parth4337**
deploy/tesetnet/bin.sh
condensed dnf statements, added installation of epel repo and removed installation of screen command
changed variable from $HOME_DIRECTORY to $HOME
Added entry to add changes to $PATH in $HOME/.bashrc profile to make them consistent across all terminal sessions

docs/testnet/onomy-testnet-docs
faucet.md and pre-installation.md
changed a bit of technical details and improved docs

### CODE REVIEW

File: deploy/testnet/bin.sh [hunk 1]
```diff
@@ -1,80 +1,59 @@
...
-export PATH=$HOME_DIRECTORY/.cargo/bin:$PATH
+export PATH=$HOME/.cargo/bin:$PATH
 cargo version
 
 echo "----------------cloning_repository-------------------"
-
-GRAVITY_DIR=$HOME_DIRECTORY/gravity
-ONOMY_DIR=$HOME_DIRECTORY/go/onomy
 git clone -b v0.0.0-20210915184851-orch-nomarket https://github.com/onomyprotocol/cosmos-gravity-bridge.git $GRAVITY_DIR
 git clone -b dev https://github.com/onomyprotocol/onomy.git $ONOMY_DIR
```
Comment (dzmitryhil) [inline deploy/testnet/bin.sh]: not dev branch here,  we need to change it to the v0.0.1 tag

File: docs/testnet/onomy-testnet-docs/pre-installation.md [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
+`
+export PATH=$PATH:$HOME/binaries/
+`
+
+### <a name="compileInstall"></a> 2. Compile Onomy chain binaries 
+If you have Fedora 34 or Red Hat Enterprise Linux 8 and you want to make binaries yourself, then follow these steps
+1. Clone Onomy repo. (You might need to install git using `dnf install git`).
 ```
-sudo yum install -y git
 git clone -b dev https://github.com/onomyprotocol/onomy.git
```
Comment (sunnyk56) [inline docs/testnet/onomy-testnet-docs/pre-installation.md:31]: we need to change branch dev to v0.0.1 tag

File: deploy/testnet/bin.sh [hunk 3]
```diff
@@ -1,80 +1,59 @@
...
- dnf -y install moreutils
- dnf -y install wget
- dnf -y install screen
-
-HOME_DIRECTORY=$HOME
+echo "-----------Installing_dependencies---------------"
+dnf -y update
+dnf -y copr enable ngompa/musl-libc
+dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
```
Comment (sunnyk56) [inline deploy/testnet/bin.sh:9]: I don`t think we need these ones, we can remove these
```
dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
```

Comment (Parth4337) [reply deploy/testnet/bin.sh:9]: For you to install moreutils, you need these. I checked and we need moreutils for sponge command used in master-node/init.sh script

File: deploy/testnet/bin.sh (hunk 1 above)
Comment (Parth4337) [reply deploy/testnet/bin.sh]: Changing this to v0.0.1. Also, this is another instance where we can use latest tag. We don't want to update a script everytime we update our binaries

**Review by dzmitryhil — APPROVED** [review]

**Review by sunnyk56 — APPROVED** [review]

**Pull request merged**